src/Form/Data/EntityRow.php

```diff
@@ -4,7 +4,7 @@
 
 namespace EMS\CoreBundle\Form\Data;
 
-use EMS\CoreBundle\Entity\EntityInterface;
+use EMS\SubmissionBundle\Entity\EntityInterface;
```

Comment:
I do not like this, this is the EntityRow for all entity datatables. And It now uses the SubmissionBundle entityInterface allways?

Reply:
And the src/Entity/EntityInterface.php  is extension of EMS\SubmissionBundle\Entity\EntityInterface

Reply:
It extends a common interface now